### ISSUE TITLE
New Commits in Repo Break Previous Code

### ISSUE THREAD

**Issue by dfan** [facebookresearch/SlowFast#544]
With the commit prior to the unsupervised additions, I am able to easily get 70+% top-1 accuracy using the `Kinetics/SLOW_8x8_R50.yaml` finetuning config. But with the new unsupervised code, I can barely get 40% top-1 accuracy on Kinetics-400 using the same config and training command.

There are a number of bugs I had to fix for the code to run, but I think there may be more. Could the authors consider carefully examining the new codebase for bugs that were introduced with recent code?
- PYAV issue (see https://github.com/facebookresearch/SlowFast/pull/541)
- LR scaling no longer works. With  [99a655bd533d7fddd7f79509e3dfaae811767b5c](https://github.com/facebookresearch/SlowFast/commit/99a655bd533d7fddd7f79509e3dfaae811767b5c), the call to [assert_and_infer_cfg()](https://github.com/facebookresearch/SlowFast/blob/c4ecfa1cfb204b2b3d8a0f6f44807e851303bd11/slowfast/config/defaults.py#L960) was removed from `run_net.py`. Thus LR scaling doesn't work as nowhere else in the code scales the LR by the number of shards.

I am also unable to reproduce the results of the MoCo unsupervised model trained with p=2 for 200 epochs, but there are larger issues if supervised R50 no longer works on Kinetics either.

**Comment by dfan** [comment]
My commit in https://github.com/facebookresearch/SlowFast/pull/541 had a small bug as noted by XinyuSun's comment. After adding that fix on top of my commit, I am able to achieve proper results on K400 for both supervised and unsupervised. Hope this helps anyone else running into this issue
